=== modified file 'a/src/lightdm-gtk-greeter.c'
--- a/src/lightdm-gtk-greeter.c	2018-02-19 02:07:35 +0000
+++ b/src/lightdm-gtk-greeter.c	2018-02-21 10:03:36 +0000
@@ -95,7 +95,7 @@
 static GtkImage     *power_icon;
 
 static const gchar *DEFAULT_LAYOUT[] = {"~spacer", "~spacer", "~host", "~spacer",
-                                        "~session", "~a11y", "~clock", NULL};
+                                        "~session", "~a11y", "~clock", "~power", NULL};
 
 static const gchar *POWER_WINDOW_DATA_LOOP = "power-window-loop";           /* <GMainLoop*> */
 static const gchar *POWER_WINDOW_DATA_RESPONSE = "power-window-response";   /* <GtkResponseType> */

